src/Zend/Form/Element.php

```diff
@@ -573,7 +573,7 @@ public function setValue($value)
      * @param  string $key
      * @return void
      */
-    protected function _filterValue(&$value, &$key)
+    protected function _filterValue(&$value, $key)
```

Comment:
`$key` isn't used at all here, but PHP8 complained that it needed to be passed as a reference (this is used as a callback function, so PHP is calling it internally)